.eslintrc

```diff
@@ -1,10 +1,33 @@
 {
-  "env":  {
-    "node": true,
-    "mocha": true
+  "env": {
+      "browser": true,
```

Comment:
is this correct?

Reply:
I used the Eslint config of my boilerplate, it does not effect, but i will remove it.